addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -217,25 +217,40 @@
 </template>
 
 <template id="s_mail_block_event" name="Event">
-    <div class="o_mail_snippet_general o_mail_block_event s_event bg-300">
+    <div class="o_mail_snippet_general s_mail_block_event bg-100 pt32 pb32" data-snippet="s_mail_block_event">
```

Comment:
I think this is going to require an upgrade script.

Reply:
even if it passed the ci/upgrade_enterprise test ?

Reply:
Maybe best to ask the _specialist_: @KangOl

Reply:
Hi @KangOl Could you look at this and tell us if it needs an upgrade script please ? Thanks :)

Reply:
I don't think anything is needed.
But I'm clearly not the *specialist* of the snippets.
Maybe @rdeodoo can answer the question.

Reply:
I don't think anything is needed here regarding the migration.
The easiest way to figure is just to create a 15 or 15.3 DB, than create a mailing with this snippet, then migrate to master on this branch and see.

![image](https://user-images.githubusercontent.com/30048408/167159562-60c7d6b4-f41c-4dc8-8897-8ab5cabc0509.png)

From there, that's not really my responsability to say this is ok or not but AFAIC this is OK.
The only issue I could think of, is that:
- Someone created a mass mailing not yet send, planed for later and kept that default image
- Migrated to the next version
- Sent the mass mailing post migration.

Seems louch enough to not be worth considering.